src/util/pelz_json_parser.c

```diff
@@ -240,6 +375,38 @@ int decrypt_parser(cJSON * json, charbuf * key_id, charbuf * data)
     pelz_log(LOG_ERR, "Missing required JSON key: dec_data_len.");
     return (1);
   }
+  /* This will be re-instated when signatures and certificates are no longer optional in JSON requests
```

Comment:
See previous comments ... I think this commented out code should move into the functionality that validates a signed request. This should also negate the need to duplicate it for both the encrypt_parser() and decrypt_parser() functionality.